proof/refine/ARM/Reply_R.thy

```diff
@@ -574,4 +574,77 @@ lemma replyRemoveTCB_invs':
               simp: cteCaps_of_def)
   done
 
+context begin interpretation Arch .
+
+lemma pspace_relation_reply_update_conc_only:
+  "\<lbrakk> pspace_relation s s'; s x = Some (Structures_A.Reply reply); s' x = Some (KOReply reply');
+     reply_relation reply new\<rbrakk>
+       \<Longrightarrow> pspace_relation s (s'(x \<mapsto> (KOReply new)))"
```

Comment:
Maybe use `ps` instead of `s` in this lemma? It can be a bit confusing otherwise, since `s` normally refers to a complete state.

Also, it's very minor but the final line is normally indented less than this. The other lemmas following this also have similarly inconsistent indentation

Reply:
> Maybe use `ps` instead of `s` in this lemma? It can be a bit confusing otherwise, since `s` normally refers to a complete state.

Ah, yes, you are right! I think I typed `sc_replies_relation` a bit too often :D

Reply:
Will fix the indentation, too.